jbmc/unit/java_bytecode/java_trace_validation/java_trace_validation.cpp

```diff
@@ -147,16 +147,24 @@ TEST_CASE("java trace validation", "[core][java_trace_validation]")
   {
     goto_tracet trace = make_test_trace(valid_symbol_expr, valid_constant);
     REQUIRE_NOTHROW(check_trace_assumptions(
-      trace, namespacet(symbol_tablet()), messaget(), true));
+      trace,
+      namespacet(symbol_tablet()),
+      messaget(),
+      true,
+      validation_modet::EXCEPTION));
   }
 
   SECTION("check_trace_assumptions fail with an invalid steps - invalid lhs")
   {
     goto_tracet trace = make_test_trace(invalid_symbol_expr, valid_constant);
     REQUIRE_THROWS_AS(
       check_trace_assumptions(
-        trace, namespacet(symbol_tablet()), messaget(), true),
-      std::runtime_error);
+        trace,
+        namespacet(symbol_tablet()),
+        messaget(),
```

Comment:
🚫 This constructor is deprecated.

Reply:
Ah, yes, it is. There's a testing util I've replaced it with.